src/Raven.Studio/typescript/components/common/RadioToggle.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import classNames from "classnames";
+
+interface ToggleProps {
+    className?: string;
+}
+
+// TODO: add whole logic
```

Comment:
Is it reported?

Reply:
https://issues.hibernatingrhinos.com/issue/RavenDB-20254/Add-logic-to-the-RadioToggle-react-component